Classes/Domain/Model/Book.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TTN\Books\Domain\Model;
+
+final class Book
```

Comment:
Please add some PHPDoc with a short description (and an `@author` if you'd like to take the credit).